### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

**Comment by tkrille** [comment]
Do we really want to change hashCode/equals in this way?

**Comment by wallner** [comment]
I don't think we should.

**Comment by dacrome** [comment]
Maybe I do not have to this way, but when I do not change equals the equalsverifier test is failing

**Comment by wallner** [comment]
Then fix the test please.

**Comment by dacrome** [comment]
I removed the version bump of the `equalsverifier` dependency. I will create an own PR for that.

File: pom.xml [hunk 1]
```diff
@@ -117,7 +117,7 @@
         <dependency>
             <groupId>org.spockframework</groupId>
             <artifactId>spock-core</artifactId>
-            <version>1.0-groovy-2.4</version>
+            <version>1.1-groovy-2.4-rc-2</version>
```
Comment (tkrille) [inline pom.xml]: This can be updated to `1.1-groovy-2.4-rc-3` now.

File: pom.xml [hunk 2]
```diff
@@ -91,14 +91,14 @@
...
-            <version>2.23.1</version>
+            <version>2.23.2</version>
         </dependency>
 
         <!-- JSON -->
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.8.1</version>
+            <version>2.8.4</version>
```
Comment (tkrille) [inline pom.xml]: This can be updated to `2.8.5` now.

File: pom.xml [hunk 3]
```diff
@@ -91,14 +91,14 @@
         <dependency>
             <groupId>org.glassfish.jersey.connectors</groupId>
             <artifactId>jersey-apache-connector</artifactId>
-            <version>2.23.1</version>
+            <version>2.23.2</version>
```
Comment (tkrille) [inline pom.xml]: This can be updated to `2.24` now.

**Comment by dacrome** [comment]
Updated.

**Comment by tkrille** [comment]
Thanks

**Pull request merged**